src/card/card.jsx

```diff
@@ -12,12 +12,46 @@ const Card = React.createClass({
     };
   },
 
+  getDefaultProps() {
```

Comment:
I swear this is the last one :sweat_smile: Can you please move to to after propTypes?